lalrpop/src/build/mod.rs

```diff
@@ -322,11 +322,17 @@ fn report_content(content: &Content) -> term::Result<()> {
     canvas.write_to(&mut stdout)
 }
 
-fn emit_module_attributes<W: Write>(
+pub fn emit_module_attributes<W: Write>(
     grammar: &r::Grammar,
     rust: &mut RustWrite<W>,
 ) -> io::Result<()> {
-    rust.write_module_attributes(grammar)
+    rust!(rust, "#[cfg_attr(rustfmt, rustfmt_skip)]");
```

Comment:
Something feels fishy about this. This attribute is going to apply to the *next* module thing that gets emitted ... oh, I see, and the for loop below is intentionally converting `#![foo]` into `#[foo]`?